dflag/dynjson_test.go

```diff
@@ -20,6 +20,16 @@ var defaultJSON = &outerJSON{
 	},
 }
 
+var defaultJSONTwo = &outerJSON{
+	FieldInts:   []int{2, 3, 4, 5},
+	FieldString: "non-empty",
```

Comment:
I changed it to `JSONValue2` because I guess the second value shouldn't be default. Lemme know if I understand this correctly. Also, what's the naming convention for this kind of variables? Thx.

Reply:
I meant you did change the ints to be different from the other entry, bit the FieldString had same value; to check the serialization/deserialization isn't accidentally returning same/mixed up values; having unique values for all test values is useful

Reply:
i guess it's not tested per se if I read the test correctly the values are changed in the test anyway so, sgtm